app/lib/my_module.rb

```diff
@@ -0,0 +1,34 @@
+module MyModule
+  def count_and_paste_result_into_test_table
+    #creation_hash_with_key_question_id_and_value_number_of_answers
+    test_answers = self.answers
+                                .where( answers: {checked: 'true'} )
+                                .group_by(&:question_id)
+    array1 = []
+    array1 = test_answers.map {|k,v| [k, v.length]}
+    array_with_question_id_and_answers_number = array1.flatten
+    hash_with_question_id_and_answers_number = Hash[*array_with_question_id_and_answers_number]
+    hash_with_number_of_tries_bigger_1 = hash_with_question_id_and_answers_number.select {|k,v| v > 1}
+
+    #creation_array_with_question_id_that_has_true_answ
+    true_answers = Answer.belongs_to_current_test(self.id)
+                          .with_true
+                          .joins(:answer_setting)
+                          .where( answer_settings: {rigth: 'true'} )
+
+    array_question_id_with_true_answ = []
+    true_answers.each do |t_a|
```

Comment:
you can call map for that. 
like 

some_name = true_answers.map{|t_a| t_a.question_id}